### PR TITLE
add_s01e04_normal_themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Thematic document: https://docs.google.com/spreadsheets/d/1C3x4tk_95TNpMOfK0yYgK424GmI4M0ZXuT5gwmvvCTQ/edit#gid=146772076

### CODE REVIEW

File: code_schemes/all_locations_s01e04.json [hunk 1]
```diff
@@ -3,7 +3,86 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "other_theme",
+      "NumericValue": 7,
+      "StringValue": "other_theme",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-9e3af59d",
+      "CodeType": "Normal",
+      "DisplayText": "enact stiff laws",
```
Comment (DanielMwendwa) [inline code_schemes/all_locations_s01e04.json:65]:
```suggestion
      "DisplayText": "enact strict laws",
```

File: code_schemes/all_locations_s01e04.json [hunk 2]
```diff
@@ -3,7 +3,86 @@
...
+      "NumericValue": 8,
+      "StringValue": "enact_stiff_laws",
```
Comment (DanielMwendwa) [inline code_schemes/all_locations_s01e04.json:67]:
```suggestion
      "StringValue": "enact_strict_laws",
```

File: code_schemes/bungoma_s01e04.json [hunk 3]
```diff
@@ -1,8 +1,72 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "educate the community",
+      "NumericValue": 1,
+      "StringValue": "educate_the_community",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-9e3af59d",
+      "CodeType": "Normal",
+      "DisplayText": "enact stiff laws",
```
Comment (DanielMwendwa) [inline code_schemes/bungoma_s01e04.json:17]:
```suggestion
      "DisplayText": "enact strict laws",
```

File: code_schemes/bungoma_s01e04.json [hunk 4]
```diff
@@ -1,8 +1,72 @@
...
+      "NumericValue": 2,
+      "StringValue": "enact_stiff_laws",
```
Comment (DanielMwendwa) [inline code_schemes/bungoma_s01e04.json:19]:
```suggestion
      "StringValue": "enact_strict_laws",
```

File: code_schemes/kiambu_s01e04.json [hunk 5]
```diff
@@ -1,8 +1,80 @@
...
+      "NumericValue": 1,
+      "StringValue": "educate_the_community",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-9e3af59d",
+      "CodeType": "Normal",
+      "DisplayText": "enact stiff laws",
+      "NumericValue": 2,
+      "StringValue": "enact_stiff_laws",
```
Comment (DanielMwendwa) [inline code_schemes/kiambu_s01e04.json:19]:
```suggestion
      "CodeType": "Normal",
      "DisplayText": "enact strict laws",
      "NumericValue": 2,
      "StringValue": "enact_strict_laws",
```

**Review by DanielMwendwa — APPROVED** [review]

File: code_schemes/all_locations_s01e04.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/all_locations_s01e04.json:65]: Will keep as is because this has since been labeled and consider the suggestion next time. Thanks

**Pull request merged**